util/pmconfig.py

```diff
@@ -41,24 +46,42 @@ def set_address(old_address, new_address, stage):
     stage.writeParamsToFlash(new_address)
 
 
-def auto_conf(address, stage):
-    # Not accurate enough, use manual configuration instead
+def spc_auto_conf(address, stage):
+    """
+    Set spc parameter with automatic configuration procedure.
+    """
     logging.info("Running auto configuration on address %s." % address)
     stage.runAutoConf(address)
-    time.sleep(3)
+    time.sleep(3)  # wait for autoconf to finish
     stage.writeParamsToFlash(address)
     logging.info("SPC value %s saved to flash.", stage.readParam(address, 11))
 
 
-def manual_conf(address, stage):
+def spc_manual_conf(address, stage, steps=200):
+    """
+    Set spc parameter with manual configuration procedure.
+    This procedure uses a larger move to determine the parameter and is move accurate than the
+    automatic configuration procedure.
+    """
     axname = [name for name, num in stage._axis_map.items() if num == address][0]  # get axis name from number
+
+    # Move to a position in the middle of the axis
+    stage.reference({'x'}).result()
+    stage.moveAbsSync({'x': 0.005})
```

Comment:
That's really ad-hoc to this hardware. I think that alternatively, you could do nothing, and expect the user to have move to a good position already. For now, I'm fine with this, if this simplifies the usage, but please put a big TODO or WARNING here.
Also, if the device is not yet configured, I wonder how well the referencing and move by 5 mm will be reliable.